docs/en/operation/Server-Logging.md

```diff
@@ -0,0 +1,162 @@
+
+You can modify the `log4j.properties` found under the `${ALLUXIO_HOME}/conf/log4j.properties` to modify logging levels.
+
+### Modifying Logging at Runtime
+
+Alluxio shell comes with a `logLevel` command that allows you to get or change the log level of a
+particular class on specific instances.
+
+The sytnax is `alluxio logLevel --logName=NAME [--target=<master|worker|host:port>] [--level=LEVEL]`,
+where the `logName` indicates the logger's name, and `target` lists the Alluxio masters or
+workers to set. If parameter `level` is provided the command changes the logger level, otherwise it
+gets and displays the current logger level.
+
+For example, this command sets the class `alluxio.heartbeat.HeartbeatContext`'s logger level to
+DEBUG on master as well as a worker at `192.168.100.100:30000`.
+
+```bash
+alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=master,192.168.100.100:30000 --level=DEBUG
+```
+
+And the following command gets all workers' log level on class `alluxio.heartbeat.HeartbeatContext`
+```bash
+alluxio logLevel --logName=alluxio.heartbeat.HeartbeatContext --target=workers
+```
+
+## Remote Logging
+
+### Overview
+
+Alluxio supports sending logs to a remote log server over the network. This feature can be useful
+to system administrators who have to perform the task of log collection. With remote logging, the
+log files, e.g. master.log, worker.log, etc. on all Alluxio servers will be readily available on
+a designated and configurable directory on the log server.
+
+### Deploying The Log Server
+
+#### Start Log Server
+
+On the log server, execute the following command.
+
+```bash
+$ ./bin/alluxio-start.sh logserver
+```
+
+#### Stop the Log Server
+
+```bash
+$ ./bin/alluxio-stop.sh logserver
+```
+
+### Configuring Alluxio Processes to use the Log Server
+
+By default, remote logging is not enabled. To enable Alluxio remote logging, you can set
+environment variables or modify Alluxio's `log4j.properties`
+
+There is no requirement on where the log server can be run, as long as the other Alluxio servers
+have network access to it. In our example, we run the log server on the same machine as a master.
+
+#### Enable Remote Logging with Environment Variables
+
+The three environment variables `ALLUXIO_LOGSERVER_HOSTNAME`, `ALLUXIO_LOGSERVER_PORT` and
+`ALLUXIO_LOGSERVER_LOGS_DIR` control Alluxio remote logging behavior.
+
+Suppose the hostname of the log server is `AlluxioLogServer`, and the port is `45010`.
+In `conf/alluxio-env.sh`, add the following lines:
+
+```bash
+ALLUXIO_LOGSERVER_HOSTNAME=AlluxioLogServer
+ALLUXIO_LOGSERVER_PORT=45010
+ALLUXIO_LOGSERVER_LOGS_DIR=/tmp/alluxio_remote_logs
+```
+
+#### Enable Remote Logging with `log4j.properties`
+
+You can also choose to modify the `${ALLUXIO_HOME}/conf/log4j.properties` file on the machines where
+your masters or workers reside to add an appender which will send log entries to the log server. A
+sample configuration is provided below:
```

Comment:
With this configuration, how do we change the logserver logs dir (`/tmp/alluxio_remote_logs` above)